src/dashboards/components/RefreshContext.tsx

```diff
@@ -14,9 +13,14 @@ import {
 } from 'src/shared/actions/autoRefresh'
 import {getCurrentDashboardId} from 'src/dashboards/selectors'
 
+// Utils
+import {createDateTimeFormatter} from 'src/utils/datetime/formatters'
+import {incrementDate} from 'src/shared/utils/dateTimeUtils'
+
 export const AutoRefreshContext = createContext(null)
 
-const DEFAULT_TIME_AHEAD = '01:00'
+// default time increment is 1 hour
+const DEFAULT_TIME_AHEAD = 1
```

Comment:
```suggestion
const oneHourInTheFuture = 1
```

or 
```ts
const ONE_HOUR_IN_THE_FUTURE = 1
```
I think the idea that this is a default is implied and somewhat apparent in this variable. I think it's more documenting to name it after what it's value represents, rather than what it does. Letting the reader know this is one hour in the future, rather than a default value, which is kind of apparent from using it, makes it so the reader doesn't have to scroll up to find out what the default value is. If you can encode a comment inside a variable name and remove the need for a comment, that's probably a well-named variable.

I think it's fine to not name this like a constant in all caps. when I think of a constant, I think of something that's constant, like the date texas became a state, or the lat/long of los angeles. This is more relative to the system we're using, but it's not a big deal.